indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorStateTest.java

```diff
@@ -2689,6 +2690,55 @@ public Duration getEmissionDuration()
     replayAll();
   }
 
+  @Test
+  public void testMaxAllowedStopsWithStopTaskCountPercent()
+  {
+    LagAggregator lagAggregator = createMock(LagAggregator.class);
+    AutoScalerConfig autoScalerConfig = new LagBasedAutoScalerConfig(
```

Comment:
Looking at the various test cases, I think it might make sense to add a `Builder` for the `LagBasedAutoScalerConfig` class. Not a blocker for this PR, though.